add adguard for windows v8 section (#785)
AdguardTeam/KnowledgeBase: docs/adguard-for-windows-8/adguard-for-windows-8.md

---
title: AdGuard for Windows v8
sidebar_position: 1
---

:::info

This article describes the beta version of AdGuard for Windows v8.0, a comprehensive ad blocker that protects your device at the system level.

> This is a beta release and still under development. If you’d like to try it, download the [beta version of AdGuard for Windows v8](https://agrd.io/adguard_for_windows_8_beta).

Reviewer: ```suggestion
This is a beta version that is still under development. If you’d like to try it, download the [beta version of AdGuard for Windows v8](https://agrd.io/adguard_for_windows_8_beta).
```

:::

AdGuard for Windows is an easy-to-use tool that blocks annoying banners, popups, video ads, and trackers. But that’s not all: it helps you save traffic, load web pages faster, and protect your computer from undesirable and potentially malicious websites. AdGuard also protects you from phishing websites and keeps your kids away from inappropriate content.

## [Home screen](/adguard-for-windows-8/home-screen/)

A toggle to enable or disable AdGuard protection, along with icons for activating Ad blocking, Tracking protection, DNS protection, and Annoyance blocking. Settings can be accessed through the left-side menu.

## [Protection](/adguard-for-windows-8/protection/)

> Settings to configure Ad blocking, Tracking protection, DNS protection, Annoyances blocking, phishing protection, Parental control, and user scripts

Reviewer: ```suggestion
Settings to configure *Ad blocking*, *Tracking protection*, *DNS protection*, *Annoyances blocking*, phishing protection, *Parental control*, and userscripts
```
